CMakeLists.txt

```diff
@@ -17,58 +17,127 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(pluginlib REQUIRED)
 find_package(rclcpp REQUIRED)
-find_package(Boost REQUIRED)
+find_package(Boost REQUIRED COMPONENTS headers)
 
 ##############################################################################
 # Build
 ##############################################################################
 
-add_library(real_time_circular_buffer INTERFACE)
-target_include_directories(real_time_circular_buffer INTERFACE
-  include
-  ${Boost_INCLUDE_DIRS})
+add_library(realtime_circular_buffer
+  INTERFACE
+    include/filters/realtime_circular_buffer.h
+    include/filters/realtime_circular_buffer.hpp
+)
+target_link_libraries(realtime_circular_buffer
+  INTERFACE
+    Boost::boost
+)
+target_include_directories(realtime_circular_buffer
```

Comment:
Instead of repeating this call, consider using a for loop and giving a list of targets to set up the includes for.
This would reduce code duplication.

Reply:
address [here](https://github.com/ros/filters/pull/70/commits/7ea64fd9b191bbe60fd57f5595de20ce25882e57)